src/App.js

```diff
@@ -26,24 +27,21 @@ const alex = {
   isMarried: false,
 };
 
+const Persons = [misha, olya, alex];
+
 const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">My wife&apos;s name is Natasha</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">My husband&apos;s name is Maksym</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    {
+      Persons.map(person => {
+        return <Person
+          name= {person.name}
```

Comment:
read and apply keys when mapping inside jsx: https://reactjs.org/docs/lists-and-keys.html#embedding-map-in-jsx